app/assets/stylesheets/_navbar.scss

```diff
@@ -53,12 +53,42 @@ $header-border-color: #ddd;
     float: left;
   }
 
-  .nav-user {
+  .nav-right {
     float: right;
+  }
+
+  .nav-search {
+    float: left;
+    line-height: 60px;
+
+    input {
+      border: 0;
+      display: inline-block;
+      font-size: 0.9em;
+      height: 30px;
+      margin: 0;
+    }
+
+    .list_search {
```

Comment:
Selector should have depth of applicability no greater than 2, but was 3<br>Selector `list_search` should be written in lowercase with hyphens